src/Utils/TerminusPackage.php

```diff
@@ -51,20 +49,12 @@ public function getInstallDir()
         return $this->install_dir;
     }
 
-    /**
-     * @return mixed Returns the version of Terminus currently installed
-     */
-    public function getInstalledVersion()
-    {
-        return str_replace('* ', '', $this->getOutdatedInfo()['versions']);
-    }
-
     /**
      * @return string Latest version of Terminus according to Composer
```

Comment:
Change the comment -- this method is now responsible for returning the initial version of Terminus to download from GitHub. After that, self:update is used to. get to the latest version.

Reply:
Could also rename it to `getInitialDownloadVersion()` or something like that.